uniflow/flow/config.py

```diff
@@ -134,6 +134,70 @@ class TransformCopyConfig(TransformConfig):
     )
     model_config: ModelConfig = field(default_factory=lambda: {})
 
+class TransformForClassificationOpenAIGPT3p5Config(TransformConfig):
+    flow_name: str = "TransformOpenAIFlow"
+    model_config: ModelConfig = field(
+        default_factory=lambda: OpenAIModelConfig(
+            model_name="gpt-3.5-turbo-1106",
+            model_server="OpenAIModelServer",
+            num_call=1,
+            temperature=0,
+            response_format={"type": "text"},
+        )
+    )
+    prompt_template: PromptTemplate = field(
+        default_factory=lambda: PromptTemplate(
+            instruction="""
+            Does the text mention any cutting-edged technology applications, any new technology methods, or any new area of innovations? If yes, return the names of each technology in a list of strings as the answer. If no, return an empty list.
```

Comment:
qq: Based on my empirical result, LLM usually is not reliable to return a `list` format. How reliable can LLM generate list response based on your experiment.

Reply:
It's quite robust for now. I tried 6 different inputs, and it can always generate correct list and return empty list for gpt3.5, and return json for gpt4